Models/DTOs/ProjectInputDTO.cs

```diff
@@ -25,7 +25,7 @@ public class ProjectInputDto
 
         [Required]
         [AllowedValues(Constants.PROJECT_PRICETYPE_PERHOUR, Constants.PROJECT_PRICETYPE_FIXED, ErrorMessage ="Price type is invalid.")]
-        public string PriceType { get; set; }
+        public string? PriceType { get; set; }
```

Comment:
You made it optional NIGGAAAAAAAAAA!

Reply:
Yes, because this was part of his task, the price type is no longer required and it is `fixed` by default

Reply:
Yeah i know, But it is still used `[Required]`.
It should be made Constant and without using [Required]. 
If it is `fixed` by default, why is it required?

Reply:
Yeah, he should have deleted `Required`, although it doesn't affect the wanted outcome, it is still a skill issue.
But wdy by making it "Constant"?

Reply:
It is still a skill issue 😂. 

He can make it easly by using `Constants.PROJECT_PRICETYPE_FIXED`, i guess we have it in `Constants` class.
But as a default PriceType is `fixed` right now so we don't have this change.